index.html

```diff
@@ -5,61 +5,170 @@
+            <li class="col-header"><h3>Skilss </h3></li>
+            <li class="list-items">Codekit</li>
+            <li class="list-items">Guthub</li>
+            <li class="list-items">Codepen</li>
+            <li class="list-items">Gitlab</li>
+            <li class="list-items">Terminal</li>
+            
+        </ul>
+    </section>
+
+    <div class="icons-section">
+
     
+    <section> 
+
+    <img src="images/git.svg" alt="">
+    <img src="images/linkedin.svg" alt="">
+    <img src="images/othericon.svg" alt="">
+    <img src="images/twitter.svg" alt="">
+    <img src="images/micon.svg" alt="">
+    </section>
+    <section class="arrow-bottom">
+        <img class="arrow-bottom" src="images/elementright.svg" alt="">
+    </section>
+    
+   </div>
+
+
+   <section class="my-work">
+    <p>MY RECENT WORK</p>
+
+    <hr>
+</section > 
+
+<section class="post-container">
+<section class="white-bg"> </section>
+<section class="posts">
+    <h2>Multi-Post Stories</h2>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+         when an unknown printer took a standard dummy text.</p>
+
+    <P class="tags">css <span class="span"></span>  html <span class="span"></span>bootstrap <span class="span"></span> Ruby </p>
+
+        <button class="see-projects-btn">See Project</button>
+</section>
 
         
+</section>
+
+    <section class="post-container">
+        <section class="white-bg"> </section>
+        <section class="posts">
+            <h2>Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                 when an unknown printer took a standard dummy text.</p>
+        
+            <P class="tags">css <span class="span"></span>  html <span class="span"></span>bootstrap <span class="span"></span> Ruby </p>
+        
+                <button class="see-projects-btn">See Project</button>
+                </section>
+        
+                
+            </section>
+
+
+            <section class="post-container">
+                <section class="white-bg"> </section>
+                <section class="posts">
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                         when an unknown printer took a standard dummy text.</p>
+                
+                    <P class="tags">css <span class="span"></span>  html <span class="span"></span>bootstrap <span class="span"></span> Ruby </p>
+                
+                        <button class="see-projects-btn">See Project</button>
+                        </section>
+                
+                        
+                    </section>
+
+
```

Comment:
- [ ] I think it would be a good practice, and to have a clear codebase if your container has only one `section` tag, and then in your `section` tag you can put your `div`, `hx`,... tags and not have many `section` tags.